node/pkg/reporter/bigtablewriter.go

```diff
@@ -109,39 +109,41 @@ func BigTableWriter(events *AttestationEventReporter, connectionConfig *BigTable
 						errC <- err
 					}
 				case msg := <-sub.Channels.VAAQuorumC:
-					colFam := "QuorumState"
-					mutation := bigtable.NewMutation()
-					ts := bigtable.Now()
-
-					b, marshalErr := msg.Marshal()
-					if marshalErr != nil {
-						logger.Error("failed to marshal VAAQuorum VAA.")
-					}
-					mutation.Set(colFam, "SignedVAA", ts, b)
-
-					// filter to see if this row already has the signature.
-					filter := bigtable.ChainFilters(
-						bigtable.FamilyFilter(colFam),
-						bigtable.ColumnFilter("SignedVAA"))
-					conditionalMutation := bigtable.NewCondMutation(filter, nil, mutation)
-
-					rowKey := MakeRowKey(msg.EmitterChain, msg.EmitterAddress, msg.Sequence)
-					err := tbl.Apply(ctx, rowKey, conditionalMutation)
-					if err != nil {
-						logger.Error("Failed to write persistence info to BigTable",
-							zap.String("rowKey", rowKey),
-							zap.String("columnFamily", colFam),
-							zap.Error(err))
-						errC <- err
-					}
-					publishResult := pubsubTopic.Publish(ctx, &pubsub.Message{
-						Data: []byte(b),
-					})
-					if _, err = publishResult.Get(ctx); err != nil {
-						logger.Error("Failed getting GCP PubSub publish reciept",
-							zap.String("rowKey", rowKey),
-							zap.Error(err))
-					}
+					go func() {
+						colFam := "QuorumState"
```

Comment:
I think it's fine because the work being done is very little and go can handle thousands of goroutines easily. The pubsub should batch requests internally and therefore not cause too much I/O. The bigtable writing is a bit more concerning and ideally we would batch those writes ourselves, but this is all code that will be rewritten soon anyways, so probably not worth the effort.

Reply:
I'm just trying to make the code better here, addressing a production issue, not make it perfect :)